tests/tests/test-staking/test-staking-locks.ts

```diff
@@ -543,39 +550,61 @@ describeDevMoonbeam("Staking - Locks - bottom and top delegations", (context) =>
   it("should be set for bottom and top list delegators", async function () {
     await expectOk(
       context.createBlock(
-        [...topDelegators].map((account, i) =>
-          context.polkadotApi.tx.parachainStaking
-            .delegate(alith.address, MIN_GLMR_DELEGATOR + 1n * GLMR, topDelegators.length + 1, 1)
-            .signAsync(account)
-        )
+        [...topDelegators].map((account, i) => {
+          // add a tip such that the delegation ordering will be preserved, e.g. the first txns sent
+          // will have the highest tip
+          let tip = BigInt(topDelegators.length - i + 1) * MILLIGLMR;
+          return context.polkadotApi.tx.parachainStaking
+            .delegate(alith.address, MIN_GLMR_DELEGATOR + 1n * GLMR, i + 1, 1)
+            .signAsync(account, { tip });
+        })
```

Comment:
This can be simplified by:

```typescript
let tip = 1 * GLMR;
await expectOk(
      context.createBlock(
        [...topDelegators].map((account, i) => {
          // add a tip such that the delegation ordering will be preserved, e.g. the first txns sent
          // will have the highest tip
          return context.polkadotApi.tx.parachainStaking
            .delegate(alith.address, MIN_GLMR_DELEGATOR + 1n * GLMR, i + 1, 1)
            .signAsync(account, { tip: tip-- });
        })
      )
    );
```